frontend/scss/pages/_p-landingpage-show.scss

```diff
@@ -4,6 +4,79 @@
   }
 }
 
+.home {
+  .o-hours {
+    margin-top: 24px;
+  }
+
+  .m-showcase-block {
+    background-color: unset;
+  }
+  
+  .m-home-header {
+    .m-media__img {
+      height: 675px;
+    }
+  }
+
+  .home-top {
+    margin-top: 25px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 20px;
+    border-bottom: 1px solid #E6E6E6;
```

Comment:
See comment: https://github.com/art-institute-of-chicago/artic.edu/pull/111/files#r1334716934